scene/main/node.h

```diff
@@ -757,6 +757,8 @@ Error Node::rpc_id(int p_peer_id, const StringName &p_method, VarArgs... p_args)
 #define ERR_READ_THREAD_GUARD_V(m_ret)
 #endif
 
+#define SET_NAME_TO_CASE_ADJUSTED_CLASS(nodePtr) nodePtr->set_name(adjust_name_casing(nodePtr->get_class()))
```

Comment:
I would prefer a method instead of a macro:

```c++
_FORCE_INLINE_ void set_name_to_case_adjusted_class() { set_name(adjust_name_casing(get_class())); }
```

It might make sense to name the method `reset_name()`, although this is too vague a name and might confuse someone into thinking the name will reset to something like `@Node@123`.

Reply:
In either case it should be:
```suggestion
#define SET_NAME_TO_CASE_ADJUSTED_CLASS(m_node_ptr) m_node_ptr->set_name(adjust_name_casing(m_node_ptr->get_class()))
```